pkg/server/services/event.go

```
@@ -0,0 +1,90 @@
+		_, err := e.client.EventsV1().Events(event.Namespace).Create(ctx, event, metav1.CreateOptions{})
+		return err
+	case types.UpdateRequestAction:
+		_, err := e.client.EventsV1().Events(event.Namespace).Update(ctx, event, metav1.UpdateOptions{})
+		return err
+	default:
+		return fmt.Errorf("unsupported event action %s", eventType.Action)
+	}
+}
+
+func (e EventService) RegisterHandler(handler server.EventHandler) {
+	return
```

golangci-lint (1.64.8)

S1023: redundant `return` statement

(gosimple)